lib/helper/WebDriverIO.js

```diff
@@ -1422,6 +1639,29 @@ class WebDriverIO extends Helper {
   }
 
   /**
+   * Waits for a specified number of elements on the page
+   */
+  waitNumberOfVisibleElements(locator, num, sec) {
```

Comment:
Yes, you can use XPath for this, but it will be not so descriptive as this method. Also using special XPath force to store two locators for one element (looks like bad practice)